app/views/admin/cms/collections/_form.html.erb

```diff
@@ -32,6 +34,12 @@
   </div>
 
   <div>
+    <%= image_tag(@collection.image.thumb) if @collection.image.present? %>
```

Comment:
This is a temporary solution, I need to either migrate all the existing file to the image column or I need to add behaviour that will use MiniMagick to retrieve the files via the url on s3, resize them to thumbnails and display.